routers/user.controller.js

```diff
@@ -0,0 +1,245 @@
+// const userModel = require("../database/modules/user.model");
```

Comment:
Please remove all of that commented code. If you'll need it someday - you'll find it and paste, it's not a big deal